tests/functional/utils.py

```diff
@@ -64,3 +65,20 @@ def recreate_censored_content(content: str, matched_string: str) -> str:
     """Applies ggshield censoring to any occurrence of `matched_string` inside
     `content`. Returns the censored string."""
     return content.replace(matched_string, recreate_censored_string(matched_string))
+
+
+# Use this in iac tests until iac hooks are added to the install command
+# Meaning they are added to install.py
+def create_local_hook(
+    path: Path,
+    hook_type: str,
+) -> None:
+    """Create hook directory (if needed) and pre-commit/pre-push file."""
+    hook_dir_path = path / ".git/hooks"
+    hook_dir_path.mkdir(parents=True, exist_ok=True)
+    hook_path = hook_dir_path / hook_type
+
+    with hook_path.open("w") as f:
+        f.write("#!/bin/sh\n")
```

Comment:
```suggestion
        f.write("#!/usr/bin/env sh\n")
```
Not entirely sure this is the better way. It would avoid edge cases where there is no /bin/sh file